sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/ng/SqlBase.g4

```diff
@@ -838,6 +839,8 @@ OUTPUTFORMAT: 'OUTPUTFORMAT';
 INPUTDRIVER: 'INPUTDRIVER';
 OUTPUTDRIVER: 'OUTPUTDRIVER';
 DATABASE: 'DATABASE' | 'SCHEMA';
+DATABASES: 'DATABASES';
```

Comment:
@hvanhovell Herman, yeah.. they are synonyms. I was not sure, If we just define it as one Token as say "DATABASES" , we can still refer to it from the nonReserved rule as 
"DATABASES" ?